vllm/_custom_ops.py

```diff
@@ -163,6 +163,18 @@ def fused_add_rms_norm(input: torch.Tensor, residual: torch.Tensor,
     torch.ops._C.fused_add_rms_norm(input, residual, weight, epsilon)
 
 
+# Advance a step on GPU for existing inputs for a multi-step runner.
```

Comment:
nit: should be a docstring (standard practice in python, allows autodoc tools to pick it up)

Reply:
changed to docstring